cmd/aznum2words-webapp/aznum2words-webapp.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	"github.com/egasimov/aznum2words/cmd/aznum2words-webapp/api"
+	"github.com/egasimov/aznum2words/cmd/aznum2words-webapp/config"
+	"github.com/egasimov/aznum2words/cmd/aznum2words-webapp/handler"
+	"github.com/egasimov/aznum2words/cmd/aznum2words-webapp/router"
+	"github.com/jessevdk/go-flags"
+	"github.com/joho/godotenv"
+	"github.com/labstack/gommon/log"
+	"os"
+)
+
+func main() {
+	var err error
+	_, err = flags.Parse(&config.Conf)
+	if err != nil {
+		panic(err)
+	}
+
+	initEnvVars()
+	config.LoadConfig()
+
+	// Create an instance of our handler which satisfies the generated interface
+	var azNum2WordsApi = new(api.AzNum2WordsApi)
+
+	r := router.New()
+	h := handler.NewHandler(azNum2WordsApi)
+	h.Register(r)
+
+	// And we serve HTTP until the world ends.
+	r.Logger.Fatal(r.Start(config.Conf.Host + ":" + config.Conf.Port))
+}
+
+func initEnvVars() {
+	var env = os.Getenv("ENVIRONMENT")
```

Comment:
Maybe create other file to hold all key names that need be read from OS environment variables to minimize typo errors when referring it and being one place to look at when config key values are used improperly(maybe missed or typo in the key name).

such as: 
File: constants.go
Content: 
```
// Config values read from OS environment
const (
	DEPLOY_ENV_KEY           = "DEPLOY_ENV"
)
```